src/commands/workflow/get.ts

```diff
@@ -108,30 +120,96 @@ export default class WorkflowGet extends BaseCommand<typeof WorkflowGet> {
      * Workflow steps table
      */
 
-    const steps = workflow.steps.map((step, index) => ({ ...step, index }));
+    const { steps } = this.flattenSteps(workflow.steps);
 
     ux.table(steps, {
       index: {
         header: "Steps",
-        get: (step) => step.index + 1,
+        get: (stepOrBranch) => stepOrBranch.key,
       },
       ref: {
         header: "Ref",
         minWidth: 18,
+        get: (stepOrBranch) =>
+          stepOrBranch.type === "branch" ? "" : stepOrBranch.ref,
       },
       type: {
         header: "Type",
         minWidth: 12,
+        get: (stepOrBranch) =>
+          stepOrBranch.type === "branch" ? "" : stepOrBranch.type,
       },
       summary: {
         header: "Summary",
-        get: (step) => Workflow.formatStepSummary(step),
+        get: (stepOrBranch) =>
+          stepOrBranch.type === "branch"
+            ? Workflow.formatBranchSummary(stepOrBranch)
+            : Workflow.formatStepSummary(stepOrBranch),
       },
       conditions: {
         header: "Conditions",
-        get: (step) =>
-          step.conditions ? Conditions.formatConditions(step.conditions) : "-",
+        get: (stepOrBranch) => {
+          if (stepOrBranch.type === Workflow.StepType.IfElse) return "-";
+
+          if (stepOrBranch.type === "branch" && stepOrBranch.isDefault) {
+            return "Default branch";
+          }
+
+          if (!stepOrBranch.conditions) return "-";
+
+          return Conditions.formatConditions(stepOrBranch.conditions);
+        },
       },
     });
   }
+
+  // Flattens a nested list of steps and branches for a workflow tree into a
+  // simple list + also includes some formatting logic for the step/branch
+  // "level" values
+  private flattenSteps(
+    steps: Workflow.WorkflowStepData[],
+    totalSteps = 0,
+    parentBranches: string[] = [],
+  ): { steps: WorkflowStepOrBranchWithKey[]; totalSteps: number } {
```

Comment:
I made this function a private method on the get command class (rather than a function in the workflow helpers lib) because it felt very specific to this one command's use case.